src/plugins/dev_tools/public/dev_tools_icon.tsx

```diff
@@ -6,18 +6,13 @@
 import React from 'react';
 import { EuiButtonIcon } from '@elastic/eui';
 import { CoreStart } from 'opensearch-dashboards/public';
-import { DEFAULT_NAV_GROUPS } from '../../../core/public';
 
 export function DevToolsIcon({ core, appId }: { core: CoreStart; appId: string }) {
   return (
     <EuiButtonIcon
       aria-label="go-to-dev-tools"
       iconType="consoleApp"
       onClick={() => {
-        /**
-         * This is a workaround in 2.16, once devTools being refactor to a drawer, we can remove the setCurrentNavGroup line.
-         */
-        core.chrome.navGroup.setCurrentNavGroup(DEFAULT_NAV_GROUPS.dataAdministration.id);
```

Comment:
would be nice if @SuZhou-Joe can confirm the removing of this line.

Reply:
@Hailong-am Thanks for the fix, I like the change. And thanks @ruanyl for helping to review.